Automattic/wp-calypso
# Editor: Introduce Gutenberg in Calypso behind feature flag

#### Changes proposed in this Pull Request

* Add a new editor path that displays Gutenberg without an iframe - this only appears when a new AB test is in the "on" variant.

This will allow us to more easily test ongoing work to fix Calypso's global styles. By checking for specific things that need to be fixed, we may be able to better prioritize somethings over others (like Buttons may not be necessary to fix immediately as they seem to be okay, but the FormRadio and FormCheckbox do need to be fixed).

Ongoing discussion here p9oQ9f-iN-p2.

##### AB test

The AB test will always bucket people into the "off" variant. This is correct and desired because we want to be able to easily test Gutenberg in Calypso as we make new style changes without introducing changes to the front-end.

#### Testing instructions



* Ensure that by default you are bucketed into the "off" variant of the new AB test
* When in the off variant, ensure that you cannot reach the new editor through navigation (i.e., ensure that the "write" button and post lists all navigate you to the iframed editor)
* When in the "on" variant, ensure that you are directed towards Gutenberg in Calypso

Fixes #
